nexus/db-queries/src/db/lookup.rs

```diff
@@ -760,6 +760,15 @@ lookup_resource! {
     primary_key_columns = [ { column_name = "id", rust_type = Uuid } ]
 }
 
+lookup_resource! {
+    name = "SledInstance",
+    ancestors = [],
+    children = [],
+    lookup_by_name = true,
```

Comment:
Don't think I saw anywhere actually doing a lookup by name?

Reply:
We're not, that's true! Good catch.